app/models/user.rb

```diff
@@ -9,6 +11,35 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
 
+  # Returns the hash digest of the given string.
```

Comment:
Since this method doesn't actually relate at all to _users_, please move it to another file. Even a new file in app/models called **digest.rb** which just has this single method would be fine. Maybe place it in a module for namespacing?